dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/util/ValidationUtils.java

```diff
@@ -40,6 +56,12 @@ public boolean isValidBase64Key(String value) {
 		}
 	}
 
+	/**
+	 * Check if the given date is in the range of [now - retentionPeriod ... now], exclusive
```

Comment:
it is a closed interval isn't it? so inclusive? or else we should use parenthesis

Reply:
it's exclusive, so I use `()`. But in fact it's using `Instant.now()` twice, so I think it's off by some milliseconds anyway...

Reply:
OK, after discussion on slack, it's `inclusive`... Sorry for the mixup.